app/helpers/success_alert_helper.rb

```diff
@@ -0,0 +1,15 @@
+module SuccessAlertHelper
+  def access_and_permissions_granted_params(application_id, granting_access:, user: current_user)
+    if granting_access
+      {
+        message: "Access granted",
+        description: access_granted_message(application_id, user),
+      }
+    else
+      {
+        message: "Permissions updated",
+        description: message_for_success(application_id, user),
```

Comment:
Could we possibly change the name of `message_for_success` to something like `permissions_updated_message` to make this more readable without much context (and more in line with the better name you chose: `access_granted_message`)?